### PR TITLE
[Sixgill-195] DVE split into separate package

### PR DESCRIPTION
**Pull request by tamarsix**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Based on a call with Matt we have decided to split the Sixgill-Darkfeed package into a separate Package that will include the DVE solution only

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details: Sixgill-Darkfeed won't include the DVE feed
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by Noy-Maimon — CHANGES_REQUESTED** [review]
Thank you for your contribution! I changed some minor logics to fit our code conventions. other than that, please see the required changes in the comments I left.
Plus, please add an README file to the integration, see more details here https://xsoar.pan.dev/docs/documentation/readme_file

**Comment by tamarsix** [comment]
@Noy-Maimon Hi, I made the required changes

File: Packs/Cybersixgill-DVE/Integrations/CybersixgillDVEEnrichment/CybersixgillDVEEnrichment.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      secret: false
+    deprecated: false
+    execution: false
+    description: Returns information for each CVE in the input list
+    name: cybersixgill-cve-enrich
+    outputs:
+      - contextPath: Sixgill.CVE.Overview
+        description: Overview of the CVE enrich information
+        type: String
+      - contextPath: Sixgill.CVE.rawJSON
```
Comment (Noy-Maimon) [inline Packs/Cybersixgill-DVE/Integrations/CybersixgillDVEEnrichment/CybersixgillDVEEnrichment.yml:48]: Please note that these outputs are being stored as a dict in out context data - which is the incidents cache. therefore, you should store them under a **meaningful** key so you could reach them in a logical way later if they will be needed (if they are not needed, then they shouldn't be stored). Currently, you are storing them all as one dict under `Sixgill.CVE.rawJSON`, and later you will not be able to access each key via demisto. So in order to be able to use them, you should store each under their key (like the value of the CVE is stored under `Sixgill.CVE.value`). There is a simple command for creating the outputs for all the json elements in our demisto-sdk:` json-to-outputs`. See more info here https://xsoar.pan.dev/docs/integrations/context-and-outputs#integration-yml. Plus, if the user wishes to see the whole rae json of the command, her can use a flag raw_response=True when running the command.
Let me know if I can help in any way.

**Review by Noy-Maimon — CHANGES_REQUESTED** [review]

**Comment by Noy-Maimon** [comment]
@tamarsix It seems like there was a problem with the last commit (note the number of changes files) - please revert it :)

**Review by Noy-Maimon — APPROVED** [review]

**Review by Noy-Maimon — APPROVED** [review]

**Pull request merged**